src/Frontend/Core/Engine/Model.php

```diff
@@ -303,4 +304,9 @@ private static function unserializeArrayContent(array $array, string $key): arra
 
         return $array;
     }
+
```

Comment:
Is this needed, I think we want to move away from the static methods

Reply:
No idea, don't even know what this method is meant to be doing

Reply:
Fixed ed40a9650